src/updateResourceId/index.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Return the URN of the resource following the next format:
+ *      '<"oma"|"ext"|"x">:<ObjectID>:<ObjectVersion>@<LWM2MVersion>'
+ * 
+ * If 'ObjectVersion' and/or 'LWM2MVersion' equal to '1.0', value is omitted. 
+ *  
+ * More info: https://github.com/NordicSemiconductor/lwm2m-types-js/blob/994cf2502fd9a50462e62c7ae3a58b714b4327bd/src/type-generation/createURN.ts#L9
+ * 
+ */
+export const getURN = async (id: string): Promise<string> => {
+  try {
+    const element = await import(
+      `../../node_modules/@nordicsemiconductor/lwm2m-types/types/${id}`
```

Comment:
Instead of relying on the internal structure of the package, export and use the [`createURN` method](https://github.com/NordicSemiconductor/lwm2m-types-js/blob/99e8a7a78fd2cce2354e9637d356199a7e087d82/src/type-generation/createURN.ts) from `@nordicsemiconductor/lwm2m-types`